docs/reference/quantity.md

```diff
@@ -789,6 +805,22 @@ the denominator.
     troubleshooting guide to understand the risks.  If you end up using it anyway, consider adding
     a brief comment to explain why it's OK in your use case.
 
+!!! tip
+    If your inputs have the _same dimension_, you most likely want to convert them to the same unit
+    --- their _common unit_ --- before dividing.  This removes the need for `unblock_int_div` in the
+    most common cases, and is generally much safer.  See
+    [`divide_in_common_units()`](#divide-in-common-units) below for more details.
+
+#### `divide_in_common_units()` {#divide-in-common-units}
+
+The `divide_in_common_units()` utility takes two `Quantity` inputs, `a`, and `b`, whose dimension is
+the same.  It first converts each input to their [common
+unit](../discussion/concepts/common_unit.md), and then performs regular division.  The result
+will be a dimensionless and unitless number (or, after [#185], a `Quantity`).
```

Comment:
Can we return the value in a `Quantity` of the common unit?

1. This is a new API so nobody will break.
2. We don't have to fix anyone that uses this API prior to #185.
3. If we return a raw number, we then have to infer what the common unit is.

As an example, if I add this to the test cases we have above:

```c++
    EXPECT_THAT(divide_in_common_units(hours(8), minutes(60)), SameTypeAndValue(8));
    EXPECT_THAT(divide_in_common_units(hours(8), hours(1)), SameTypeAndValue(8));
```

Those are two very different `8`s!  This is the kind of thing that a units library should be trying to prevent... 😁

Reply:
New comment is good.  I probably still would've made the error because I was really thinking the new API was doing something else (I was thinking it was more of a replacement for `unblock_int_div` but it definitely is a more narrow use case).